coins/zen.json

```diff
@@ -93,8 +93,8 @@
     "txfee": 0.0004,
 
     "explorer": {
-        "txURL": "https://explorer.zen-solutions.io/tx/",
-        "blockURL": "https://explorer.zen-solutions.io/block/",
+        "txURL": "https://explorer.zensystem.io//tx/",
+        "blockURL": "hhttps://explorer.zensystem.io//block/",
```

Comment:
It looks like there are double `/` in both the links and the `blockURL` has `hhttps` instead of `https`.